syntax = "proto3";

package span_data_container;

option go_package = ".;proto_gen";

// SpanDataContainer is a wrapper for storing a span and its resource together.
// This is used internally by the ingestion service to persist spans and their
// associated resources together in BadgerDB.
//
// Note: This is an internal-only proto, not part of the gRPC service API.
message SpanDataContainer {
  bytes span_bytes = 1;
  bytes resource_bytes = 2;
}
